frontend/src/app/shared/components/datepicker/single-date-modal/single-date.modal.html

```diff
@@ -10,37 +10,14 @@
 
   <div class="spot-modal--body form -vertical">
     <div class="op-datepicker-modal--toggle-actions-container">
-      <div class="form--field">
-        <label class="form--label">
-          {{text.scheduling.title}}
-        </label>
-        <div class="form--field-container">
-          <spot-toggle
-            [options]="schedulingOptions"
-            [value]="scheduleManually"
-            [name]="'scheduling'"
-            [(ngModel)]="scheduleManually"
-            (ngModelChange)="changeSchedulingMode()"
-            data-qa-selector="op-datepicker-modal--scheduling-action"
-          ></spot-toggle>
-        </div>
-      </div>
-
-      <div class="form--field">
-        <label class="form--label">
-          {{ text.ignoreNonWorkingDays.title }}
-        </label>
-        <div class="form--field-container">
-          <spot-toggle
-            [options]="ignoreNonWorkingDaysOptions"
-            [value]="ignoreNonWorkingDays"
-            [name]="'include-non-working-days'"
-            [(ngModel)]="ignoreNonWorkingDays"
-            (ngModelChange)="changeNonWorkingDays()"
-            data-qa-selector="op-datepicker-modal--include-non-working-days"
-          ></spot-toggle>
-        </div>
-      </div>
+      <op-datepicker-scheduling-toggle
+        [scheduleManually]="scheduleManually"
+        (schedulingChanged)="scheduleManually = $event"
```

Comment:
It's a rather big change but it'd be nice if this worked with ngModel as a ControlValueAccessor

Reply:
It's a bit weird to have to repeat the CVA, but I agree that it makes using them a bit nicer. Done